web/content/docs/benchmarks/thermo-mechanics/thermomechanics.md

```diff
@@ -31,9 +31,6 @@ $$
 \end{equation}
 $$
 
-The relative error between the numerical simulation and the analytical solution is 9.2 $\cdot$ 10\uexp{-13}.
-
-Kolditz, Olaf, Uwe-Jens G�rke, Hua Shao, and Wenqing Wang, eds. Thermo-hydro-mechanical-chemical processes in porous media: benchmarks and examples. Vol. 86. Springer Science & Business Media, 2012.
-
-
+The relative error between the numerical simulation and the analytical solution is $9.2 \cdot 10^{-13}$.
 
+Kolditz, Olaf, Uwe-Jens Görke, Hua Shao, and Wenqing Wang, eds. Thermo-hydro-mechanical-chemical processes in porous media: benchmarks and examples. Vol. 86. Springer Science & Business Media, 2012.
```

Comment:
Name schema in citation is inconsistent: Either: 'first name last name, ...' or 'last name first name'.